l10n_it_fatturapa_in/wizard/link_to_existing_invoice.py

```diff
@@ -16,6 +16,33 @@ def get_invoice_obj(self, fatturapa_attachment):
         xml_string = fatturapa_attachment.get_xml_string()
         return fatturapa_v_1_2.CreateFromDocument(xml_string)
 
+    def compute_xml_amount_untaxed(self, DatiRiepilogo):
+        amount_untaxed = 0.0
+        for Riepilogo in DatiRiepilogo:
+            amount_untaxed += float(Riepilogo.ImponibileImporto)
+        return amount_untaxed
+
+    def compute_xml_amount_tax(self, DatiRiepilogo):
+        amount_tax = 0.0
+        for Riepilogo in DatiRiepilogo:
+            amount_tax += float(Riepilogo.Imposta)
+        return amount_tax
+
+    def set_einvoice_amount(self, fattura):
+        amount_untaxed = self.compute_xml_amount_untaxed(
+            fattura.DatiBeniServizi.DatiRiepilogo)
+        amount_tax = self.compute_xml_amount_tax(
+            fattura.DatiBeniServizi.DatiRiepilogo)
+        amount_total = float(
+            fattura.DatiGenerali.DatiGeneraliDocumento.
+            ImportoTotaleDocumento or 0.0)
+
+        self.invoice_id.update({
+            'e_invoice_amount_untaxed': amount_untaxed,
+            'e_invoice_amount_tax': amount_tax,
+            'e_invoice_amount_total': amount_total,
+        })
+
```

Comment:
@robyf70 sarebbe meglio non duplicare i metodi, ma farne uno solo e usarlo da entrambi i wizard

Reply:
> ma farne uno solo

Cioè mettere i metodi in un unico posto e chiamare `set_einvoice_amount` da entrambi i wizard

Reply:
@eLBati Lo spostiamo dentro `account.invoice`?

Reply:
@eLBati In realtà abbiamo anche `compute_xml_amount_tax` e `compute_xml_amount_untaxed` che potenzialmente andrebbero messi in un'unico posto

Reply:
@eLBati Potremmo eventualmente fare una classe `Mixins` dove appoggiare questi metodi ed ereditarli da entrambi i wizard, che ne dici?

Reply:
> Lo spostiamo dentro account.invoice?

Ok

> In realtà abbiamo anche compute_xml_amount_tax e compute_xml_amount_untaxed che potenzialmente andrebbero messi in un'unico posto

Esatto

> Potremmo eventualmente fare una classe Mixins dove appoggiare questi metodi ed ereditarli da entrambi i wizard, che ne dici?

Mi sembra eccessivo: ci serve solo poter chiamare un metodo da 2 wizard diversi.
I metodi fanno operazioni sulla fattura, quindi secondo me `account.invoice` va bene

Reply:
@eLBati Ho appena pushato, dai uno sguardo alla patch e dimmi che ti sembra